packages/sync-repo-settings/package.json

```diff
@@ -12,13 +12,14 @@
   "files": [
     "build/src"
   ],
+  "bin": "build/src/bin/sync-repo-settings.js",
```

Comment:
to reduce confusion, I like to create a `cli.ts` here so it's super obvious which file does what.